server.js

```diff
@@ -41,22 +41,18 @@ app.get('/throw-an-error', (request, response) => {
 
 //Weather Route for json data
 app.get('/weather', (request, response) => {
-  let cityName = request.query.city_name;
-  // let latitude = request.query.lat;
-  // let longitude = request.query.lon;
-  // let findCity = weatherData.filter(city => city.city_name === cityName || city.lat || city.lon === latitude && longitude);
-  let findCity = weatherData.filter(city => city.city_name === cityName);
-  // console.log(findCity);
-  let groomedData = findCity[0].data.map(day => new Forecast(day));
-
-  response.send(groomedData);
-});
-
-
-//Catch Route Always Must be the last route in file and can control the message
-app.get('*', (request, response) => {
-  response.status(404).send('UH OH! Something\'s Wrong!');
-
+  try {
+    let cityName = request.query.city_name;
```

Comment:
- Good work on the updates 👍 
- We need to: 
Create an API endpoint of /weather that processes a GET request that contains **lat, lon** and searchQuery information.